sound/soc/intel/boards/Kconfig

```diff
@@ -466,7 +467,7 @@ config SND_SOC_INTEL_SOF_RT5682_MACH
 	depends on I2C && ACPI
 	depends on (SND_SOC_SOF_HDA_LINK && (MFD_INTEL_LPSS || COMPILE_TEST)) ||\
 		   (SND_SOC_SOF_BAYTRAIL && (X86_INTEL_LPSS || COMPILE_TEST))
-	depends on SND_HDA_CODEC_HDMI
+	depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC
```

Comment:
@libinyang Here I'd merge this with SOF_HDA_LINK on L468. If this mach driver needs the audio codecs, then no need to test with HDA_LINK here.

Reply:
@kv2019i OK, I will move SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC to L468